src/config/adapters/dhis_highcharts/series/index.js

```diff
@@ -5,6 +5,7 @@ import getType, { isDualAxis } from '../type';
 import { CHART_TYPE_PIE, CHART_TYPE_GAUGE } from '../type';
 import { getFullIdAxisMap, getAxisIdsMap, hasExtraAxisItems } from '../seriesItems';
```

Comment:
Is `hasExtraAxisItems` still used in this file?

Reply:
Thanks, fixed.